frontend/src/app/modules/log-in/log-in-page/log-in-page.component.sass

```diff
@@ -0,0 +1,219 @@
+@import url('https://fonts.googleapis.com/css?family=Montserrat')
+@import url('https://fonts.googleapis.com/css?family=Open Sans')
```

Comment:
ad them globaly

Reply:
Done as you suggested

Reply:
I have encountered some very bizarre behavior when I add this style specifically
@import url('https://fonts.googleapis.com/css?family=Open Sans') into the global.sass
![image](https://github.com/BinaryStudioAcademy/bsa-2023-leetwars/assets/57751436/e025b3e1-1f2f-4c39-8a8b-4c169a7306df)
I tried to look up for what's the problem but can't find the solution or a hint, can you give me a hint please?

Reply:
I will try to install those fonts as package and see the results

Reply:
First issue resolved, but another with the sass file bloated after I include global is still a mystery for me

Reply:
fixed the issue